### ISSUE TITLE
Registration & Profile Page Updates - Part II

### ISSUE THREAD

**Issue by RishiRajSahu** [appirio-tech/connect-app#3277]
1. Registration Page: **Update Subtitle on Screen 1**
- [x] Update the subtitle on Screen 1 of the registration workflow to read "We're excited to meet you!" instead of "First we need to know you a bit better"

2. Registration Page: **Update Text on Screen 2 (PIN Entry)**
- [x] Update message displayed on this page to read, "We've sent a 1-time PIN to your email. Check your inbox and enter the PIN provided. If its been a few minutes and you have not received your PIN, check your spam folder first and then request a new PIN be sent if not found."
- [x] Add sub-text under PIN Entry field to indicate that entry is case-sensitive: "PIN is case-sensitive"

3. Registration Page: **Update Language & Workflow on Screen 3 (Registration complete) of registration workflow.**
- [x] Update title subtext to read: "The World's Largest Crowdsourcing Platform" instead of "The World's Biggest Crowdsourcing Platform"
- [x] Update page content to read: "Hello [username], Your Topcoder account was successfully created." (consolidate the messaging here)
- [x] Add a button underneath page content, reading "Review Catalog" which should bring the user into the Connect Catalog
- [x] Update the existing button to read, "Go to Dashboard" which should bring the user into the 'All Projects' dashboard of Connect 

4. **Merge the "Notifications Settings" and "Account & Security" tabs** in the left sidebar into the "My Profile" Tab, as both are related to managing your personal profile and preferences.
- [x] The layout of information within the "My Profile" tab should be - 
Profile Information
Notifications Settings
Account & Security

5. Profile Page: **Lift "Required" Requirement for Saving Changes**
- [ ] On the "My Profile" page, a user should be able to edit any of the following fields, ~without having to update/complete any of the other incomplete fields:~
1. First Name
2. Last Name
3. Business phone
4. Business email
5. Company~
Add edit privileges as below per roles as mentioned below - 

**Role -** Customers
First Name
Last Name
Business phone
Business email

**Role -** Admin&Managers
First Name
Last Name
Business phone
Business email
~Client company list~

**Role -** Copilots 
First Name
Last Name
Email

**Comment by lakshmiathreya** [comment]
@RishiRajSahu 'go to Dashboard' doesnt have link to All Projects Connect Dashboard.

**Comment by vigneshTheDev** [comment]
Hi @lakshmiathreya, according to the implementation, only the word Dashboard has the link. Can you please let me know the exact issue? Is the link not visible at all or is it pointing to wrong page?

**Comment by lakshmiathreya** [comment]
@vigneshTheDev - yes only 'Dashboard' needs to have the link but it seems to have no link!

**Comment by lakshmiathreya** [comment]
@vigneshTheDev For Profile updates in Customer role, 
First Name
Last Name
Business phone
Business email 

are to be the allowed, non mandatory fields right?

I notice:
-- Country is also editable. 
-- Title is mandatory - we are not supposed to have any mandatory fields in edit Profile right?

**Comment by vigneshTheDev** [comment]
@lakshmiathreya,
> yes only 'Dashboard' needs to have the link but it seems to have no link!

I see the changes reflecting in the commit. You'll also need to merge the latest changes in accounts app here: https://github.com/appirio-tech/accounts-app/pull/316. Please let me know if you still face the issue after merging this PR

**Comment by vigneshTheDev** [comment]
> @vigneshTheDev For Profile updates in Customer role,
> First Name
> Last Name
> Business phone
> Business email
> 
> are to be the allowed, non mandatory fields right?
> 
> I notice:
> -- Country is also editable.
> -- Title is mandatory - we are not supposed to have any mandatory fields in edit Profile right?

@lakshmiathreya, please hold on for a while on this requirement. We might revert it back to make all the fields mandatory. I'm waiting on @RishiRajSahu for confirmation.

**Comment by lakshmiathreya** [comment]
Sure will wait for updates reg the final version of the edit Profile reqmt ... @RishiRajSahu @vigneshTheDev

**Comment by RishiRajSahu** [comment]
@vigneshTheDev with recent discussion with the team, we don't have to lift the required fields feature, let it be as it is, just add role wise edit privileges to profile page.

@lakshmiathreya please skip dashboard URL issue you mentioned above, actually vignesh is right that change is not in dev, will push it on monday.

**Comment by RishiRajSahu** [comment]
@lakshmiathreya the latest changes including the corrected dashboard URL have been pushed to dev.

**Comment by lakshmiathreya** [comment]
All the requirements here have been verified. However:
-- in Dev, Profile page comes up with blank fields due to Member Service issue. 
-- The list of fields to be edited in Profile edit - all fields are listed - same list of fields for all Roles.
-- There is verbiage on the PIN Entry screen which is not mentioned in the Requirements above: "Make sure you hear from us".

**Comment by RishiRajSahu** [comment]
@lakshmiathreya 
-- yeah that's known
-- aren't the remaining fields read only if user don't have privilege ?
-- It was there earlier also, and there was no mention of removing it, could you compare on dev and prod this text ?

also it would be helpful to have screenshot ?

**Comment by vigneshTheDev** [comment]
@lakshmiathreya, Profile edit privileges issue is fixed now. Could you please verify?

**Comment by lakshmiathreya** [comment]
<img width="1349" alt="Screenshot 2019-09-25 at 11 49 38 AM" src="https://user-images.githubusercontent.com/51733042/65575306-21aed700-df8d-11e9-8179-d67ba4315f31.png">
<img width="1302" alt="Screenshot 2019-09-25 at 12 06 41 PM" src="https://user-images.githubusercontent.com/51733042/65575308-21aed700-df8d-11e9-8734-7df2c25c60e1.png">
<img width="1293" alt="Screenshot 2019-09-25 at 12 10 17 PM" src="https://user-images.githubusercontent.com/51733042/65575474-88cc8b80-df8d-11e9-840d-56fb7b119cd8.png">

Please review screenshots:
1. Admin and Manager role - All fields are editable and Save is enabled. Pls clarify - Company Name in the Admin Profile stands for Client Company list?
2. Copilot and Customer Role - editable fields available as per requirement - however Save button is disabled. 
3. Country is editable in all cases. This hasnt been mentioned as a requirement.

**Comment by RishiRajSahu** [comment]
@lakshmiathreya 
1. Ignore company list, as per discussion in our last team call that's out of scope for now
2. Save button gets enabled once we made changes otherwise its disabled for no changes
3. Yes, its editable

**Comment by lakshmiathreya** [comment]
2. Save button is disabled even after changes are made for Customer and Copilot roles.

**Comment by lakshmiathreya** [comment]
@RishiRajSahu as per our conversation this is a known issue. Let's go ahead with the release. I will close the Issue after Prod testing.

**Comment by RishiRajSahu** [comment]
Pushed to prod.
